src/containers/PrimaryValidatorApi/PrimaryValidatorApiBankBlocks/index.tsx

```diff
@@ -40,7 +40,7 @@ const PrimaryValidatorApiBankBlocks: FC = () => {
 }`}
         heading="Request"
       />
-      <RequestResponseSnippet code={`{}`} heading="Response" />
+      <RequestResponseSnippet code="" heading="Response" />
```

Comment:
we want these to be the way it was before. you might have to adjust eslint settings if it's complaining

Reply:
![Screen Shot 2020-11-03 at 8 18 21 AM](https://user-images.githubusercontent.com/5943963/98011771-26a7a200-1dad-11eb-8f68-8bc667e9ba3a.png)